### PR TITLE
Switch to a handlers setup for restarting galaxy

### PR DESCRIPTION
**Pull request by mvdbeek**
Force a galaxy restart only if necessary (when a new admin has been added to galaxy.ini),
and trigger handler to restart galaxy if tools have been installed.

### CODE REVIEW

File: tasks/bootstrap_user.yml [hunk 1]
```diff
@@ -16,10 +16,17 @@
 - name: Set bootstrap user as Galaxy Admin
   lineinfile: dest={{ galaxy_config_file }} state=present insertafter="app:main" regexp="^[ ]*admin_users[ ]*=[ ]*" line="admin_users = {{ galaxy_tools_admin_user }}"  #"
   when: (create_user is defined) and create_user and (admin_users_found.msg == "")
+  notify:
+    - restart galaxy
```
Comment (bgruening) [inline tasks/bootstrap_user.yml:20]: Just out of interest is the renmaming of restart_galaxy needed to make this work?

Comment (mvdbeek) [inline tasks/bootstrap_user.yml:20]: The handler task was already called "restart galaxy", but for ansible to automatically recognize "restart galaxy" as a handler it has to be in handlers/main.yml (or be included from handlers/main.yml)

Comment (bgruening) [inline tasks/bootstrap_user.yml:20]: Thanks, I thought so just wanted to double check.

**Comment by mvdbeek** [comment]
Thanks @bgruening

**Pull request merged**